addons/l10n_es_edi_tbai/models/res_company.py

```diff
@@ -134,7 +134,8 @@ def _get_l10n_es_tbai_license_dict(self):
                 license_key = self.l10n_es_tbai_tax_agency
             else:  # production env: only one license
                 license_key = 'production'
-            return L10N_ES_TBAI_LICENSE_DICT[license_key]
+            license = L10N_ES_TBAI_LICENSE_DICT[license_key]
+            return dict(license, license_name=str(license["license_name"]))  # force translation
```

Comment:
You can use the environment to translate the value.
```suggestion
            return dict(license, license_name=self.env._(license["license_name"]))  # force translation
```

Reply:
That was my first attempt. By pylint don't like it: https://runbot.odoo.com/runbot/build/76355588

Reply:
`# pylint: disable=gettext-variable`